esdt-safe/src/lib.rs

```diff
@@ -199,6 +199,23 @@ pub trait EsdtSafe: fee_estimator_module::FeeEstimatorModule + token_module::Tok
         OptionalResult::None
     }
 
+    #[view(getFirstBatch)]
```

Comment:
Let's make this as we discussed, getBatch(batchId), it will be the same as we have on eth. It will also help if the front end will at some point want to find in what batch the user has a deposit

Reply:
Done.